metrics_batcher_test.go

```diff
@@ -109,3 +110,21 @@ func makeTs(i int) *monitoringpb.TimeSeries {
 		},
 	}
 }
+
+func TestSendReqAndParseDropped(t *testing.T) {
```

Comment:
Can you add a test where error doesn't include dropped time series? Or is that already covered by existing tests?

Reply:
The existing tests are not great tbh. I've added tests for all cases related to the code I added